src/components/Global/TokenSelector/TokenSelector.tsx

```
@@ -1,421 +1,763 @@
+        let filteredByChain = sourceBalances
+        if (selectedChainID) {
+            filteredByChain = sourceBalances.filter((balance) => {
+                const balanceChainId = String(balance.chainId)
+                return balanceChainId === selectedChainID
+            })
+        }
+
+        const filteredBalances = filteredByChain.filter((balance) => {
+            const hasSymbol = !!balance.symbol
+            const symbolMatch = hasSymbol && balance.symbol.toLowerCase().includes(lowerSearchValue)
+            const nameMatch = balance.name && balance.name.toLowerCase().includes(lowerSearchValue)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)